packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-list.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Item from './contract-type-item';
+import { localize } from '@deriv/translations';
+import { Text } from '@deriv/components';
+import classNames from 'classnames';
+import { TContractType, TList } from './ContractTypeInfo/contract-type-info.js';
```

Comment:
Good job @henry-deriv, what do you think about moving `TContractType, TContractCategory and TList` to a shared type file under ContractType folder?

Reply:
Sounds good! 👍